README.rst

```diff
@@ -615,3 +619,109 @@ Stats
 
 Stats from python-zyte-api_ are exposed as Scrapy stats with the
 ``scrapy-zyte-api`` prefix.
+
+
+Request fingerprinting
+======================
+
+The request fingerprinter class of this plugin ensures that Scrapy 2.7 and
+later generate unique `request fingerprints
+<https://docs.scrapy.org/en/latest/topics/request-response.html#request-fingerprints>`_
+for Zyte API requests based on some of their parameters.
+
+For example, a request for ``browserHtml`` and a request for ``screenshot`` are
+considered different requests. Similarly, requests with different ``actions``
+are also considered different requests.
+
+Zyte API parameters that affect request fingerprinting
+------------------------------------------------------
+
+The request fingerprinter class of this plugin generates request fingerprints
+for Zyte API requests based on the following Zyte API parameters:
+
+-   ``url`` (`canonicalized <https://w3lib.readthedocs.io/en/latest/w3lib.html#w3lib.url.canonicalize_url>`_)
+
+-   Request attribute parameters (``httpRequestBody``,
+    ``httpRequestMethod``)
+
+-   Output parameters (``browserHtml``, ``httpResponseBody``,
+    ``httpResponseHeaders``, ``screenshot``)
+
+-   Rendering option parameters (``actions``, ``javascript``,
+    ``screenshotOptions``)
+
+-   ``geolocation``
+
+The following Zyte API parameters are *not* taken into account for request
+fingerprinting:
+
+-   Request header parameters (``customHttpRequestHeaders``,
+    ``requestHeaders``)
+
+-   Metadata parameters (``echoData``, ``jobId``)
+
+
+Changing the fingerprinting of non-Zyte-API requests
+----------------------------------------------------
+
+You can assign a request fingerprinter class to the
+``ZYTE_API_FALLBACK_REQUEST_FINGERPRINTER_CLASS`` Scrapy setting to configure
+a custom request fingerprinter class to use for requests that do not go through
+Zyte API:
+
+.. code-block:: python
+
+    ZYTE_API_FALLBACK_REQUEST_FINGERPRINTER_CLASS = "custom.RequestFingerprinter"
+
+By default, requests that do not go through Zyte API use the default request
+fingerprinter class of the installed Scrapy version.
+
+
+Request fingerprinting on Scrapy 2.6.3 and earlier
+--------------------------------------------------
+
+If you use Scrapy 2.6.3 or earlier, Zyte API parameters are not taken into
+account for request fingerprinting. This can cause some Scrapy components, like
+the filter of duplicate requests or the HTTP cache extension, to interpret 2
+different requests as being the same.
+
+To avoid most issues, use automated request parameter mapping, either through
+transparent mode or setting ``zyte_api_automap`` to ``True`` in
+``Request.meta``, and then use ``Request`` properties attributes instead of
+``Request.meta`` as much as possible. Unlike ``Request.meta``, ``Request``
+attributes do affect request fingerprints in Scrapy 2.6.3 and earlier.
```

Comment:
a nice catch

Reply:
Note to self: fix typo: “properties attributes” → attributes